plugin.json

```diff
@@ -21,6 +21,13 @@
     "settings_schema": {
         "header": "The GitHub plugin for Mattermost allows users to Subscribe to notifications, stay up-to-date with reviews, see the status of your pull requests at a glance, and other common GitHub actions - directly from Mattermost.  \n \n Instructions for setup are [available here](https://www.mattermost.com/pl/default-github-plugin#configuration)",
         "settings": [
+            {
+                "key": "EnableLeftSidebar",
```

Comment:
Let's move this before `EnablePrivateRepo` as the first few setting are for the setup process.

Reply:
done.